tests/move_architecture/move_architecture.py

```diff
@@ -35,3 +35,37 @@
 }
 
 move_architecture_json = {"quantum_architecture": move_architecture_specification}
+
+deneb_architecture_specification = {
```

Comment:
Not sure what the difference is, but this is what `deneb/quantum-architecture` returns from the web service.

Reply:
There is an inconsistency: `move_atchitecture`, `deneb_architecture` and the actual architecture contains Ndonis.

Reply:
I'm not sure how to rename these accurately

- `move_architecture` was there when I started the present work. It does not match what `Deneb` returns as the `Ndonis` architecture. Someone who more fully understands the concepts could suggest if this should be renamed or merged to the newer fixture.
- There are several references to `adonis` in the test fixtures. I left these as-is, but they are inconsistent with `ndonis` used elsewhere.
- `deneb_architecture_specification` does specify `Ndonis` as the name of the architecture. The content of the fixture is what I get from `https://cocos.resonance.meetiqm.com/deneb/quantum-architecture`. Is this not accurate?

Reply:
Would it be ok to remove references to Deneb and use `ndonis` in place of it?

Reply:
@tonivanhala  yes, let's do this, Deneb is a name of a computer with 6-Donis architecture, so it will be consistent.

Reply:
So these seem to be legacy inconsistencies. `adonis` was the first `ndonis` device that has a `move` in its architecture where `deneb` is the public facing name for the `ndonis` devices with 6 qubits. I think `deneb` is the most appropriate, everything that doesn't match what the IQMProvider is obtaining, should be outdated. However, I don't know where the `move_architecture` is used for since it was already there.

Reply:
renamed `deneb` to `ndonis`

Reply:
@Aerylia adonis is 5 qubits, no resonator, it does not have `move`